audited.gemspec

```diff
@@ -15,14 +14,16 @@ Gem::Specification.new do |gem|
   gem.license     = 'MIT'
 
   gem.files         = `git ls-files`.split($\).reject{|f| f =~ /(\.gemspec)/ }
-  gem.require_paths = ['lib']
 
-  gem.add_dependency 'activerecord', '>= 4.0', '< 5.1'
-  gem.add_dependency 'rails-observers', '~> 0.1.2'
+  gem.required_ruby_version = '>= 2.3.0'
```

Comment:
This is important, because one of our applications uses lower than 2.3.0 and will not be able to update to this version of audited.